src/contrasts.jl

```diff
@@ -224,6 +266,17 @@ regression model, dummy coding leads to an intercept that is the mean of the
 dependent variable for base level.
 
 Also known as "treatment coding" (`contr.treatment` in R) or "one-hot encoding".
```

Comment:
That being said, there's one place where people might get confused which we discuss, which is that the base level for effects/sum/simple coding is different in our implementation than in R and SPSS. I think it's a good idea to include that at least.